x-pack/plugins/rollup/public/crud_app/sections/job_list/detail_panel/detail_panel.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component } from 'react';
+
+import {
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiTab,
+  EuiTabs,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import {
+  TabSummary,
+  TabTerms,
```

Comment:
From looking at the docs, I think the "histogram" group is represented by the `terms` field (https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-put-job.html). What do you think of referring to this Histogram in the UI (instead of Terms as it currently is), and add a note that it's represented by `terms` in the API?

Reply:
Never mind! My mistake. You're right. I'll add Histogram in addition to the existing Terms.